src/challenge/dto/create-challenge.dto.ts

```diff
@@ -0,0 +1,34 @@
+import { Entity, ObjectIdColumn, ObjectID, Column } from 'typeorm';
```

Comment:
This one is not needed. Challenges will be created when importing the GEdIL package, their specification is inside.